packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md

```diff
@@ -33,6 +33,8 @@
 
 * **Did you delete all unused attributes?** &mdash; As outlined in the [Build a SCIM provisioning integration](/docs/guides/build-provisioning-integration/attribute-mapping/) guide, you need to update the base attributes and mappings that correspond with the attributes supported by your application. Select **Yes** if you have completed the instructions in the **Remove the mapping** and **Delete attributes from your attribute list** sub-sections in the **Delete attributes** section of that guide.
 
+* **How large would you like your Custom Patch Batch Size to be?** &mdash; This value tells Okta how many Group membership changes to submit inside a single PATCH operation. Organizing these changes into smaller chunks improves the efficiency of updates. Enter a numerical value between 100 and 1000.
```

Comment:
Organizing these changes into smaller chunks improves the efficiency of updates. > When you organize these changes into smaller chunks, you can improve the efficiency of updates.

Reply:
suggestion implemented